test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetSpecCommandTest.cs

```diff
@@ -141,13 +147,16 @@ public void SpecCommand_WithProjectFile()
     <authors>$author$</authors>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <license type=""expression"">MIT</license>
+    <!-- <icon>icon.png</icon> -->
     <projectUrl>http://project_url_here_or_delete_this_line/</projectUrl>
-    <iconUrl>http://icon_url_here_or_delete_this_line/</iconUrl>
     <description>$description$</description>
     <releaseNotes>Summary of changes made in this release of the package.</releaseNotes>
     <copyright>$copyright$</copyright>
     <tags>Tag1 Tag2</tags>
   </metadata>
+  <files>
+    <!-- <file src=""icon.png"" target="""" /> -->
```

Comment:
Nuspec packing has 2 modes:

* globbing under current directory.
* files element

Does this change what the default generated nuspec does? 

Does it move from globbing to files mode? 

I think we should add a test that runs `nuget.exe spec` and then `nuget.exe pack` in succession.

Reply:
Not sure about `globbing under current directory.` mode. Can you elaborate?
Do I need to add more files before running `nuget.exe spec`?

Reply:
Look at this example, https://github.com/NuGet/Samples/blob/main/ContentFilesExample/authoring/ContentFilesExample.nuspec, There's no files element and everything under the `nuspec` file's folder is packed.

Reply:
Please check now. Added 2 new tests for `nuget spec` and then `nuget pack` in succession.
1. `PackCommand_NoRepo_GlobAllFiles_WithDefaultNuspec_Succeeds.`
It globs all files in it. This one doesn't have any `.net` project.
2. `PackCommand_WithRepo_WithDefaultNuspec_Succeeds`
It only pack `proj1.dll`. This one has `proj1 .net` project.

Currently I don't add `<files>..</files>` tag to avoid enabling `files element` mode by default. People will figure out.